Appium install returns a non-zero exit code with NPM7

## The problem

Appium fails to install or at least returns a non-zero exit code while installing

## Environment

* Appium version (or git revision) that exhibits the issue: 1.19.0
* Last Appium version that did not exhibit the issue (if applicable): Unsure - maybe 1.11.0
* Desktop OS/version used to run Appium: OSX
* Node.js version (unless using Appium.app|exe): v15.2.1
* Npm or Yarn package manager: npm@7.0.10
* Mobile platform/version under test: N/A
* Real device or emulator/simulator: N/A
* Appium CLI or Appium.app|exe: CLI

## Details

When running the following commands on a fresh OS Catalina install:
```
brew install node
npm install -g appium
```
it errors with
```
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /usr/local/lib/node_modules/appium/node_modules/.bin/authorize-ios
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/appium/node_modules/.bin/authorize-ios'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```

However - if I go to run appium it does appear to launch. We're installing appium through Packer so the fact that it errors but is still installed is not ideal. I'm also not 100% convinced the install is complete because of this error but I can't be sure.

Someone else saw this same behavior [here](https://stackoverflow.com/questions/64655932/bin-authorize-ios-not-found-when-installing-appium-using-npm-install-g-appium)

## Other notable things:
- Since we're doing this with packer I can say that the OS is new and the test is repeatable
- I tried downgrading node but that does not seem to have an effect until v1.11.0. It also doesn't seem like I can update it to the latest

--- a/docs/cn/contributing-to-appium/appium-from-source.md
+++ b/docs/cn/contributing-to-appium/appium-from-source.md
@@ -23,7 +23,6 @@ git clone https://github.com/appium/appium.git
 cd appium
 npm install
 npm run build
-npm run authorize-ios                # for ios only
 node .
 ```
 ```center
@@ -31,7 +30,6 @@ git clone https://github.com/appium/appium.git
 cd appium
 npm install
 npm run build # 需要gulp，往下看
-npm run authorize-ios                # 仅iOS
 node .
 ```
 
@@ -73,31 +71,6 @@ node .
 ```
 
 完整的参数列表，请参考[the server documentation](/docs/cn/writing-running-appium/server-args.md)
-
-
-#### Hacking with Appium for iOS
-
-#### 鼓捣 iOS 上的Appium
-
-为了避免启动iOS应用程序时可能出现的安全对话框，您必须通过以下两种方式之一修改 `/etc/authorization` 文件：
-
-1. 手动修改 `/etc/authorization` 文件中的 `<key>system.privilege.taskport</key>` 下的 `<allow-root>`的值为 `<true/>`。
-2. 运行以下命令，为您自动修改 `/etc/authorization` 文件：
-
-    ```center
-    sudo npm run authorize-ios
-	```
-
-
-此时，运行：
-
-```center
-rm -rf node-modules
-npm install
-npm run build
-```
-
-现在你的 Appium 实例已经准备好了。运行 `node .` 以启动Appium服务器。
 
 #### 鼓捣 Android 上的Appium
 

--- a/docs/en/contributing-to-appium/appium-from-source.md
+++ b/docs/en/contributing-to-appium/appium-from-source.md
@@ -64,29 +64,6 @@ node .
 See [the server documentation](/docs/en/writing-running-appium/server-args.md)
 for a full list of command line arguments that can be used.
 
-#### Hacking with Appium for iOS
-
-To avoid a security dialog that may appear when launching your iOS apps you'll
-have to modify your `/etc/authorization` file in one of two ways:
-
-1. Manually modify the element following `<allow-root>` under `<key>system.privilege.taskport</key>`
-   in your `/etc/authorization` file to `<true/>`.
-
-2. Run the following command which automatically modifies your
-   `/etc/authorization` file for you:
-
-    ```
-    sudo npm run authorize-ios
-	```
-
-At this point, run:
-
-```
-rm -rf node_modules && rm -rf package-lock.json && npm install
-```
-
-Now your Appium instance is ready to go. Run `node .` to kick up the Appium server.
-
 #### Hacking with Appium for Android
 
 To work on Android, make sure you have `ant`, `maven`, and `adb` installed

--- a/docs/en/drivers/ios-uiautomation.md
+++ b/docs/en/drivers/ios-uiautomation.md
@@ -55,32 +55,6 @@ Reference](#TODO).
 simulator may be open, and automated, at any given time. For multiple simulator
 support, you will need to upgrade to the [XCUITest driver](ios-xcuitest.md)).
 
-1. To allow the iOS simulator to be automated by Instruments, you need to
-   modify the authorization database for the system. Appium provides an easy
-   way to do this by installing and running an authorization script:
-
-    ```
-    sudo authorize-ios
-    ```
-
-1. By default, Instruments-based automation is limited by the inclusion of
-   a 1-second hard-coded delay between commands, implemented for obscure
-   reasons by Apple's engineers. There is a way around this limitation called
-   [instruments-without-delay](https://github.com/facebookarchive/instruments-without-delay)
-   (IWD). IWD ships with Appium for Xcode versions < 7. For 7.x and up, IWD
-   must be installed manually by the user in advance of using Appium. The way
-   to do this is as follows:
-
-    * Clone the [appium-ios-driver](https://github.com/appium/appium-ios-driver)
-      repository.
-    * Inside the repo, run the `xcode-iwd.sh` script included in the `bin` dir,
-      passing it several arguments: (1) the path to the Xcode app you are
-      using. (2) The path to the appium-instruments directory. For example:
-
-        ```
-        sh ./bin/xcode-iwd.sh /Applications/Xcode.app /Users/me/appium-instruments/
-        ```
-
 1. For best results, launch each simulator you wish to use and ensure the following:
 
     * The soft keyboard is enabled (Command+K in the Simulator app)
@@ -163,8 +137,7 @@ $HOME/Library/Logs/CoreSimulator/*
 ### Running iOS tests using Jenkins
 
 First download the `jenkins-cli.jar` and verify that the Mac successfully
-connects to Jenkins master. Ensure you've run the `authorize-ios` command
-mentioned above.
+connects to Jenkins master.
 
 ```
 wget https://jenkins.ci.cloudbees.com/jnlpJars/jenkins-cli.jar

--- a/package.json
+++ b/package.json
@@ -27,8 +27,7 @@
   },
   "main": "./build/lib/main.js",
   "bin": {
-    "appium": "./build/lib/main.js",
-    "authorize-ios": "./node_modules/.bin/authorize-ios"
+    "appium": "./build/lib/main.js"
   },
   "directories": {
     "lib": "./lib",
@@ -96,8 +95,7 @@
     "generate-docs": "node ./build/commands-yml/parse.js",
     "zip": "zip -qr appium.zip .",
     "upload": "gulp github-upload",
-    "zip-and-upload": "npm run zip && npm run upload",
-    "authorize-ios": "authorize-ios"
+    "zip-and-upload": "npm run zip && npm run upload"
   },
   "pre-commit": [
     "precommit-msg",

